components/Header.jsx

```diff
@@ -0,0 +1,295 @@
+import Link from "next/link"
+import { FormattedMessage } from "react-intl"
+
+import mastodonLogo from "../public/logos/logo-full-purple.svg"
+import Image from "next/image"
+import { useState, useEffect, useRef } from "react"
+import classNames from "classnames"
+import { locales } from "../data/locales"
+import MenuToggle from "./MenuToggle"
+import SVG from "react-inlinesvg"
+
+const useMenu = ({ navigationItems }) => {
```

Comment:
kept this in the header file since it's only used here, but it is admittedly *big*. maybe should refactor it out?